portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,16 @@
+  </head>
+  <body>
+    <div id="content">
+    <link rel="stylesheet" href="style.css">
+
```

Comment:
you can remove this extra line

Reply:
which line?

Reply:
Line 5 - you have an extra line there between the <link and <h1 tags. For ease of reading, it's good to avoid having various gaps in the code.